implementations/rust/ockam/ockam_command/src/relay/show.rs

```diff
@@ -52,16 +53,25 @@ async fn run_impl(
         )
         .await?;
 
-    println!("Relay:");
-    println!("  Relay Route: {}", relay_info.forwarding_route());
-    println!(
-        "  Remote Address: {}",
-        relay_info.remote_address_ma().into_diagnostic()?
-    );
-    println!(
-        "  Worker Address: {}",
-        relay_info.worker_address_ma().into_diagnostic()?
-    );
+    match opts.global_args.output_format {
+        OutputFormat::Plain => println!(
+            r#""
+        Relay:
+            Relay Route: {}
+            Remote Address: {}
+            Worker Address: {}
+        ""#,
+            relay_info.forwarding_route(),
+            relay_info.remote_address_ma().into_diagnostic()?,
+            relay_info.worker_address_ma().into_diagnostic()?
+        ),
+        OutputFormat::Json => println!(
+            "{}",
+            serde_json::to_string(&relay_info).map_err(|e| Report::msg(format!(
+                "Can't serialize the relay type, because: {e}"
+            )))?
+        ),
+    }
```

Comment:
While we are here, it would be great to refactor a little bit how we handle the output. We have a new standard way of doing it, like in this command:

https://github.com/build-trust/ockam/blob/e9319863147e2f7c283e57986eba5c3caba53b81/implementations/rust/ockam/ockam_command/src/relay/create.rs#L150-L159

You can use `formatdoc!` to create the plain output like this:

https://github.com/build-trust/ockam/blob/9c49e470a574d91854aa24628584eda4554aaad0/implementations/rust/ockam/ockam_command/src/node/list.rs#L162-L169

Reply:
I encounter those snippets actually, but the original code was using the normal `println` macro, so I followed it without ask any questions XD

I'll fix this now

Reply:
@adrianbenavides how about now?

Reply:
The serialization process has some cost, I prefer to not do it before check if there is a chance that I don't need it. But I understand your reason.

Reply:
I think it would be better if this layer was lazy, like if I just needed to send some clojer, and it would only run it if necessary.